### ISSUE TITLE
Can we use images as marks?

### ISSUE THREAD

**Issue by jama1017** [entronad/graphic#307]
Hi!

Thanks so much for creating this great library. I wonder if it's possible to use images as marks in plots, like this example in Vega-lite: [https://vega.github.io/vega-lite/docs/image.html#examples](https://vega.github.io/vega-lite/docs/image.html#examples)

**Comment by entronad** [comment]
Yes, in Graphic, Mark only determines how it is located, Shape determines how it is rendered. For the vega example, you can use a Point Mark, and write your own Shape, like this example:
https://github.com/entronad/graphic/blob/0cd16d1f9b0448fbd9ff699220179342da47e138/example/lib/pages/polygon_custom.dart#L8
https://github.com/entronad/graphic/blob/0cd16d1f9b0448fbd9ff699220179342da47e138/example/lib/pages/polygon_custom.dart#L436

In your Shape subclass, you can render [ImageElement](https://pub.dev/documentation/graphic/latest/graphic/ImageElement-class.html)s
